cartoframes/data/dataset.py

```diff
@@ -467,10 +471,8 @@ def download(self, limit=None, decode_geom=False, retry_times=DEFAULT_RETRY_TIME
             raise ValueError('You should provide a context and a table_name or query to download data.')
 
         # priority order: query, table
```

Comment:
we might be able to remove this comment.

Reply:
We are going to split Dataset into different classes and I think these comments will be useful for that and in any case, it will be removed